template/workflows/manifests/.github/workflows/azure-kubernetes-service.yml

```diff
@@ -59,14 +60,14 @@ jobs:
       - name: Azure login
         uses: azure/login@v1.4.6
         with:
-          client-id: ${{ secrets.AZURE_CLIENT_ID }}
-          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          client-id: ${{ .secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ .secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ .secrets.AZURE_SUBSCRIPTION_ID }}
 
       # Builds and pushes an image up to your Azure Container Registry
       - name: Build and push image to ACR
         run: |
-          az acr build --image ${{ env.AZURE_CONTAINER_REGISTRY }}.azurecr.io/${{ env.CONTAINER_NAME }}:${{ github.sha }} --registry ${{ env.AZURE_CONTAINER_REGISTRY }} -g ${{ env.ACR_RESOURCE_GROUP }} -f ${{ env.DOCKER_FILE }} ${{ env.BUILD_CONTEXT_PATH }}
+          az acr build --image ${{ .env.AZURE_CONTAINER_REGISTRY }}.azurecr.io/${{ .env.CONTAINER_NAME }}:${{ .github.sha }} --registry ${{ .env.AZURE_CONTAINER_REGISTRY }} -g ${{ .env.ACR_RESOURCE_GROUP }} -f ${{ .env.DOCKER_FILE }} ${{ .env.BUILD_CONTEXT_PATH }}
```

Comment:
anything with a `${{ env` or a `${{ secret` will not change to having a `.` in front of the variable as these are. GitHub action templates that replace using the secrets or env variables from line 38+

Reply:
This should be reverted for all the workflow files